src/request.ml

```diff
@@ -809,7 +809,9 @@ value extract_henv conf base =
     match p_getenv conf.env "manitou" with
     [ Some "off" -> conf.henv := conf.henv @ [("manitou", "off")]
     | Some _ | None -> () ];
-  }
+    match p_getenv conf.env "mod" with
+    [ Some x -> conf.henv := conf.henv @ [("mod", x)]
+    | None -> () ];  }
 ;
```

Comment:
Perfect, but can you keep the closing curly braces on its own line (like it was before) ?